ktor-client/ktor-client-core/common/src/io/ktor/client/features/websocket/WebSockets.kt

```diff
@@ -15,20 +15,22 @@ import io.ktor.util.*
 /**
  * Client WebSocket feature.
  *
- * @property pingInterval - interval between [FrameType.PING] messages.
- * @property maxFrameSize - max size of single websocket frame.
+ * @property pingIntervalMillis - interval between [FrameType.PING] messages.
+ * @property timeoutMillis - timeout to receive a [FrameType.PONG] message.
  */
 @KtorExperimentalAPI
 @OptIn(WebSocketInternalAPI::class)
 class WebSockets(
-    val pingInterval: Long = -1L,
-    val maxFrameSize: Long = Int.MAX_VALUE.toLong()
+    var pingIntervalMillis: Long = -1L,
```

Comment:
Consider keeping old names for now, we can't change them until ktor `1.4.0`

Reply:
As I understand, we should keep the old names and default values but `maxFrameSize` would still be passed as timeoutMillis?